packages/angular_devkit/build_angular/src/webpack/configs/common.ts

```diff
@@ -503,7 +503,6 @@ export function getCommonConfig(wco: WebpackConfigOptions): Configuration {
     plugins: [
       // Always replace the context for the System.import in angular/core to prevent warnings.
       // https://github.com/angular/angular/issues/11580
-      // With VE the correct context is added in @ngtools/webpack, but Ivy doesn't need it at all.
       new ContextReplacementPlugin(/\@angular(\\|\/)core(\\|\/)/),
```

Comment:
```suggestion
      new ContextReplacementPlugin(/\@angular(\\|\/)core(\\|\/)/, projectRoot, {}),
```
This should remove the need to modify the SourceMap spec test to filter the sources.

Reply:
PTAL